client/src/App.js

```diff
@@ -1,19 +1,37 @@
-import React, { Component } from "react";
+import axios from 'axios';
+import React, { useEffect } from 'react';
+import { Redirect, Route, Switch, useHistory } from 'react-router-dom';
 import NavBar from './components/NavBar/index';
 import Jumbo from './components/Jumbotron/index';
-import Container from 'react-bootstrap/Container';
-import Row from 'react-bootstrap/Row';
-import Col from 'react-bootstrap/Col';
+import { Container, Row, Col, Form, Button } from 'react-bootstrap';
```

Comment:
from react-bootstrap's documentation for [importing components](https://react-bootstrap.github.io/getting-started/introduction#importing-components):

> You should import individual components like: react-bootstrap/Button rather than the entire library. Doing so pulls in only the specific components that you use, which can significantly reduce the amount of code you end up sending to the client.